pkg/ipamd/ipamd.go

```diff
@@ -407,17 +411,18 @@ func (c *IPAMContext) nodeInit() error {
 		vpcCIDRs = c.updateCIDRsRulesOnChange(vpcCIDRs)
 	}, 30*time.Second)
 
-	if c.useCustomNetworking && c.eniConfig.Getter().MyENI != "default" {
+	eniConfigName,_ := eniconfig.GetNodeSpecificENIConfigName(ctx, c.k8sClient)
```

Comment:
Don't ignore errors.  You are proceeding with an 'unknown' eniconfig name, which is very different behaviour to what you intended (eg: the below code will go and set node labels to empty string, etc)